keys.go

```diff
@@ -548,6 +555,39 @@ func (c *Client) wrap(ctx context.Context, idOrAlias string, plainText []byte, a
 	return pt, ct, nil
 }
 
+// WrapWithKeyVersion function supports KeyVersion Details, PlainText and Cyphertext in response
+func (c *Client) WrapV2(ctx context.Context, idOrAlias string, plainText []byte, additionalAuthData *[]string) (*KeyActionResponse, error) {
+	keysActionReq := &KeysActionRequest{}
+	keyActionRes := &KeyActionResponse{}
+
+	if plainText != nil {
+		_, err := base64.StdEncoding.DecodeString(string(plainText))
+		if err != nil {
+			return keyActionRes, err
+		}
+		keysActionReq.PlainText = string(plainText)
+	}
+
+	if additionalAuthData != nil {
+		keysActionReq.AAD = *additionalAuthData
+	}
+
+	keysAction, err := c.doKeysAction(ctx, idOrAlias, "wrap", keysActionReq)
+	if err != nil {
+		return keyActionRes, err
+	}
```

Comment:
This entire block is a duplicate. Could you pls write one wrapHelper function & invoke the same function from both WrapV2(....) &   wrap(....).

Reply:
Some of the code is reusable but the number of return and return values are different

Reply:
you can return `keysAction, err` from helper function. And then return different parameters from WrapV2(....) & wrap(....).

Reply:
Not required as it is just data validation & wrap (...) will be decommissioned.

Reply:
@harshit777 pls resolve this.